trt_models:
  na: ""

pt_models:
  yolov8s_best: "/home/nvidia/fsd/models/first_yolov8s.pt"

inference_params:
  conf: 0.50
  iou: 0.70
